Revamp objects BBEs (#3899)
ballerina-platform/ballerina-distribution: examples/client-class/client_class.md

# Client class

The `client` keyword is used with class definition to define a client class. Ballerina supports defining client objects to allow a program to interact with remote network services. They are a special kind of object that contain `remote` and `resource` methods in addition to regular methods.

> Similarly class objects can be constructed using object constructor as well.

Reviewer: ```suggestion
Similarly, class objects can be constructed using an object constructor as well.
```

> The `client` keyword is used with class definition to define client class. Ballerina supports defining client objects to allow a program to interact with remote network services. They are a special kind of objects that contain `remote` and `resource` methods in addition to regular methods. It is a compile-time error to have `remote` or `resource` methods in the regular class definition.

Reviewer 2: ```suggestion
The `client` keyword is used with class definition to define a client class. Ballerina supports defining client objects to allow a program to interact with remote network services. They are a special kind of object that contain `remote` and `resource` methods in addition to regular methods. 
```
Do we need this sentence since it's not really related to client class 
`It is a compile-time error to have `remote` or `resource` methods in the regular class definition`?

Author: We can remove it

Author: Updated 02e5165034b24ddfebcb68dc31e86f3f80dd2a56
